Cargo.lock

```diff
@@ -103,6 +103,25 @@ version = "0.7.6"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "7c02d123df017efcdfbd739ef81735b36c5ba83ec3c59c80a9d7ecc718f92e50"
 
+[[package]]
+name = "ascii-canvas"
```

Comment:
We need to investigate why all these packages are being updated. It appears that we are fetching the latest versions of all dependencies from `crates.io` each time.